### DISCUSSION TITLE
Separate app sound for all apps

### DISCUSSION THREAD

**Discussion by Reinsworth** [salvogiangri/UN1CA#150] [Feature requests]
Good day to you sir Salvo, currently in ONEUI separate app sound only limited to 5 apps. In UN1CA you implemented dual messenger feature to all apps, maybe you could do the same for separate app sound so it's not limited to 5 apps if that's possible. I hope you will consider it and also thanks for your time and effort for this ROM.

**Comment by salvogiangri** [comment]
Needs to be tested, but it's there now!

<img loading="lazy" src="https://github.com/user-attachments/assets/5b4ede20-3d82-4ca6-b999-4386296c789c" width="25%"/>

**Reply by Reinsworth** [reply]
Thank you so much sir, I hope it's working fine... I would be waiting for it to be released.
